maintainers/maintainer-list.nix

```diff
@@ -4207,6 +4207,12 @@
     githubId = 131599;
     name = "Martin Weinelt";
   };
+  hexagonal-sun = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.